schematic/schemas/commands.py

```diff
@@ -59,9 +59,21 @@ def schema() -> None:  # use as `schematic model ...`
     metavar="<OUTPUT_PATH>",
     help=query_dict(schema_commands, ("schema", "convert", "output_jsonld")),
 )
+@click.option("--output_path", help="Alias for --output_jsonld")
+@click.option(
+    "--output_type",
+    "-ot",
+    type=click.Choice(["jsonld", "graph", "all"], case_sensitive=False),
+    default="jsonld",
+    help=query_dict(schema_commands, ("schema", "convert", "output_type")),
+)
 def convert(
-    schema: Any, data_model_labels: DisplayLabelType, output_jsonld: Optional[str]
-) -> None:
+    schema: Any,
+    data_model_labels: DisplayLabelType,
+    output_jsonld: Optional[str],
+    output_type: Optional[str],
+    output_path: Optional[str],
```

Comment:
I'd suggest removing `output_jsonld` as a parameter and use only `output_path` in combination with `output_type` with the latter controlling the extension

Reply:
I remember Mialy mentioning keeping this for backwards compatibility for now. Can we deprecate it in a future release?